plopfile.js

```diff
@@ -52,37 +52,7 @@ module.exports = (plop) => {
         template:
           "export { default as {{pascalCase name}} } from './components/{{pascalCase name}}';",
       },
-      // From https://github.com/bradfrost/czi-vanilla-storybook
```

Comment:
this sorting removes the useful comments and sections in the index.ts file. Updated the comment so if we want to restore the sorting, can look at the diff associated with the comment